p2p/bootstrap.go

```diff
@@ -65,18 +65,18 @@ func (s *Service) bootstrapConnect() error {
 		wg.Add(1)
 		go func(p *ps.PeerInfo) {
 			defer wg.Done()
-			defer log.Println(s.ctx, "bootstrapDial", s.host.ID(), p.ID)
-			log.Printf("%s bootstrapping to %s", s.host.ID(), p.ID)
+			defer log.Info("dialing bootstrap...", "id", s.host.ID().Pretty(), "to", p.ID.Pretty())
```

Comment:
**We need to define some conventions for logging moving forward**. Please consider removing the heavy usage of ellipses, they only really make sense in a synchronous context.

Reply:
I agree we need some convention in place and yes I will remove the ellipses.